Adds Cute Pictures

nicoleyson opened the pull request:
## Issue
https://github.com/team-dumpster-fire/policies/issues/1

## What
Resolves https://github.com/team-dumpster-fire/policies/issues/1

## Test
Works as expected if you tear up 😭 ❤️

Commit:
cuteness++;

Commit:
utilizing the cute resources

Commit:
the squirrel was a jpg </3

taiidani reviewed (dismissed):
Per the newly proposed policy 2, this PR description has not been provided with a cute image!!

...plus there are `.DS_Store` files in there. Ew.

jyncka reviewed (approved):
LGTM

Commit:
hi ryan

Commit:
I hope you're happy the animals don't have anywhere to shop.

nicoleyson commented:
@taiidani 
> Per the newly proposed policy 2, this PR description has not been provided with a cute image!! 

-> https://github.com/team-dumpster-fire/policies/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R6 
<img width="1254" alt="image" src="https://user-images.githubusercontent.com/14220781/58194219-ca64c700-7c79-11e9-9c0e-f0639d7f5ba5.png">


I addressed your review, but unwillingly. Now the animals don't have a DS store to buy their 
![image](https://user-images.githubusercontent.com/14220781/58194022-40b4f980-7c79-11e9-8707-d402f7a53b57.png) 

I hope you're pleased.

nicoleyson committed:
adds newline to eof

nicoleyson commented:
Crap, I think it didn't squash merge 😂

taiidani commented:
All of your commits, propagated for the world to see...
